File: CompSci 1/Unit 7, Math Class/MinMax/MinMax.java
import java.util.Scanner;

public class MinMax {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = 0;
        int b = 0;
        int c = 0;

        System.out.print("Enter 1st Integer --> ");
        a = sc.nextInt();
        System.out.print("Enter 2nd Integer --> ");
        b = sc.nextInt();
        System.out.print("Enter 3rd Integer --> ");
        c = sc.nextInt();

        System.out.println("\nSmallest = " + Math.min(a, (Math.min(b, c))));
        System.out.println("Largest = " + Math.max(a, (Math.max(b, c))));
    }
}

File: CompSci 1/Unit 7, Math Class/MinMax/MinMax.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    var a = 0
    var b = 0
    var c = 0

    print("Enter 1st Integer --> ")
    a = sc.nextInt()
    print("Enter 2nd Integer --> ")
    b = sc.nextInt()
    print("Enter 3rd Integer --> ")
    c = sc.nextInt()

    println("\nSmallest = " + Math.min(a, Math.min(b, c)))
    println("Largest = " + Math.max(a, Math.max(b, c)))
}
